app/Domain/Integrations/UdbOrganizerCollection.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations;
+
+use Illuminate\Support\Collection;
+
+/**
+ * @extends Collection<int, UdbOrganizer>
+ */
+final class UdbOrganizerCollection extends Collection
```

Comment:
For other collections we used a naming convention to just use the plural form. For example `Realm` and `Realms`, so for this case it should be `UdbOrganizers`